### PR TITLE
Marquee - new component

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
Marquee - new component.

## Changelog
Marquee - new component.

### CODE REVIEW

**Comment by Inbal-Tish** [comment]
@lidord-wix Are you waiting for something on my end here?

**Comment by lidord-wix** [comment]
> @lidord-wix Are you waiting for something on my end here?

I thought it was still in progress, but nevermind, I'll review it right now

File: demo/src/screens/componentScreens/MarqueeScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      </View>
+    );
+  }
+  
+  render() {
+    return (
+      <>
+        <Text text50 grey10 margin-20>Marquee</Text>
+        <View marginH-50 style={{borderWidth: 1}}>
+          <Marquee start={50} end={50} duration={9000}>
```
Comment (lidord-wix) [inline demo/src/screens/componentScreens/MarqueeScreen.tsx:21]: We can remove the `start` and `end` props here

**Review by lidord-wix — COMMENTED** [review]
I still see the same issue - when I try to render a long text in a small container I have a long waiting time until I see the text again, maybe I misuse the component but this is the way I used it: 
```
        <View bg-blue60 marginT-50 marginL-50 width={100}>
          <Marquee duration={10000} width={430} direction={'left'}>
            <Text style={{borderWidth: 1}} text50>this is a very very very very long text</Text>
          </Marquee>
        </View>
```
Am I missing something in my example?

File: demo/src/screens/componentScreens/MarqueeScreen.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/MarqueeScreen.tsx]: Done

**Comment by Inbal-Tish** [comment]
@lidord-wix Can you add your very very long test to the example on line 33 and tell me if you have an issue?

**Comment by lidord-wix** [comment]
> @lidord-wix Can you add your very very long test to the example on line 33 and tell me if you have an issue?

Yes I have this issue..
The example I attached is adding a long text to line 33 + updating the width (without it, the text splits into many lines)

**Pull request closed without merging**